student-test/assignment1/Test_validate_username.c

```diff
@@ -18,5 +18,8 @@ void test_validate_my_username()
      * TODO: Replace the line below with your code here as described above to verify your /conf/username.txt 
      * config file and my_username() functions are setup properly
      */
-    TEST_ASSERT_TRUE_MESSAGE(false,"AESD students, please fix me!");
+     const char* expected = my_username();
+     char* actual = malloc_username_from_conf_file();
+     TEST_ASSERT_EQUAL_STRING(expected,actual);
```

Comment:
@SrirajVemparala what happens to the string allocated here?

Reply:
@dwalkes : I see that the pointer to the char actual is allocated using malloc. I should be freeing the pointer after the comparison is done. 
I should add line
free(actual).